Migrate user permissions from previous release

jjmerchante opened the pull request:
The previous release included the users' permissions in the Group Django model. In the new release, they are included in the Tenant model. This code migrates the permissions for all the users.

Related to https://github.com/chaoss/grimoirelab-sortinghat/issues/849

sduenas reviewed (approved):
The code is ok but the problem that I see is the first time the user migrates. It won't be any user an any group.

jjmerchante commented:
If they are included in a tenant, they will have `readonly` permissions. I can change that to `user` permissions.

sduenas commented:
> If they are included in a tenant, they will have `readonly` permissions. I can change that to `user` permissions.

Is that true? I mean, they have those permissions but there's no entry in the database saying that they have those permissions. The table `auth_user_groups` is empty.

jjmerchante commented:
> Is that true? I mean, they have those permissions but there's no entry in the database saying that they have those permissions. The table `auth_user_groups` is empty.

I think you mean that I forgot to migrate the permissions when multitenancy is disabled. I have updated that. Additionally, I changed the default permissions to `user` when multi-tenant is enabled, to match what it was before the update.

sduenas reviewed (approved):
LGTM

sduenas commented:
> > Is that true? I mean, they have those permissions but there's no entry in the database saying that they have those permissions. The table `auth_user_groups` is empty.
> 
> I think you mean that I forgot to migrate the permissions when multitenancy is disabled. I have updated that. Additionally, I changed the default permissions to `user` when multi-tenant is enabled, to match what it was before the update.

Yes, that's what I meant.
